tests/integration/utils/fixtures.py

```diff
@@ -144,7 +144,7 @@ def vega_service():
         retain_log_files=True,
         transactions_per_block=1,
         listen_for_high_volume_stream_updates=False,
-        use_full_vega_wallet=True,
```

Comment:
Is this intentional? Think we switched from `False` -> `True` yesterday and wonder if this is a conflict.